### PR TITLE
Update ministries to reflect GAiN > unto name change

### PR DESCRIPTION
**Pull request by OzzieOrca**
[Helpscout ticket](https://secure.helpscout.net/conversation/1061869260/405976/)

Should not go to production until Feb 1

### CODE REVIEW

**Comment by wrandall22** [comment]
Is there a ticket for this? Jira or Helpscout?

**Comment by wrandall22** [comment]
We should put this in stage beforehand so they can look at it as well.

File: src/app/searchResults/searchResults.ministries.js [hunk 1]
```diff
@@ -120,9 +114,9 @@ export default [
     facet: 'ministry'
   },
   {
-    name: 'Unto™',
-    designationNumber: '1071573',
-    path: '/1071573',
+    name: 'unto™',
```
Comment (wrandall22) [inline src/app/searchResults/searchResults.ministries.js]: Should use `Unto™` as before. Helpscout ticket was updated.

File: src/app/searchResults/searchResults.ministries.js [hunk 2]
```diff
@@ -121,8 +115,8 @@ export default [
   },
   {
     name: 'Unto™',
```
Comment (OzzieOrca) [inline src/app/searchResults/searchResults.ministries.js]: Is the `™` desired? It must have blended in with the quotes when I was looking at it before and I didn't notice it.

Comment (wrandall22) [reply src/app/searchResults/searchResults.ministries.js]: I believe it is.

**Pull request merged**